Maquetado/index.html

```diff
@@ -17,6 +17,7 @@ <h1 class="mainTitle">
         Título del libro
         <strong class='bookGenre'> (género)</strong>
       </h1>
+      <div class='line'></div>
```

Comment:
This could be done with [pseudoelements](https://developer.mozilla.org/en-US/docs/Web/CSS/::after) or just a margin bottom attribute.